lib/cdo/aws/rds.rb

```diff
@@ -77,7 +77,8 @@ def self.clone_cluster(
       CDO.log.info "Done creating database cluster - #{clone_cluster_id}"
     end
 
-    def self.delete_cluster(cluster_id, max_attempts = 20, delay = 60)
+    def self.delete_cluster(cluster_id: nil, max_attempts: 20, delay: 60)
+      raise StandardError.new("CLUSTER_ID environment variable is required.") unless cluster_id
```

Comment:
@jamescodeorg points out we may need to protect against an empty CLUSTER_ID. Maybe `cluster_id.exists?` would work.

Reply:
We want to make sure to cover all the cases where the `rds_client.describe_db_clusters` call below will return all clusters (instead of returning that no such cluster exists).  The docs aren't super clear on this and my informal testing shows that nil and empty string are problematic but a string like '   ' will return an error.  `nil? || empty?` is maybe more precise (based on my current understanding) but I think `blank?` is fine too.  Up to you.

Reply:
Thanks. After reviewing [this article](https://blog.arkency.com/2017/07/nil-empty-blank-ruby-rails-difference/), I went with `.present?` (the negation of `.blank?`). Both are provided by Rails.